samples/cpp/speculative_decoding_lm/speculative_decoding_lm.cpp

```diff
@@ -2,108 +2,22 @@
-    ov::genai::GenerationConfig generation_config_greedy_constant = ov::genai::greedy();
-    {
-        generation_config_greedy_constant.num_assistant_tokens = 5;
-    }
-
-    ov::genai::GenerationConfig generation_config_multinomial_constant = ov::genai::multinomial();
-    {
-        generation_config_multinomial_constant.num_assistant_tokens = 5;
-        generation_config_multinomial_constant.num_return_sequences = 1;
-    }
-
-    ov::genai::GenerationConfig generation_config_greedy_dynamic = ov::genai::greedy();
-    {
-        generation_config_greedy_dynamic.assistant_confidence_threshold = 0.8f;
-    }
-
-    ov::genai::GenerationConfig generation_config_multinomial_dynamic = ov::genai::multinomial();
-    {
-        generation_config_multinomial_dynamic.assistant_confidence_threshold = 0.8f;
-    }
-
-    return {
-        generation_config_greedy_constant,
-        generation_config_multinomial_constant,
-        generation_config_greedy_dynamic,
-        generation_config_multinomial_dynamic,
-    };
-}
-
 int main(int argc, char* argv[]) try {
-    // Command line options
-
-    cxxopts::Options options("accuracy_sample", "Help command");
-
-    options.add_options()
-    ("n,num_prompts", "A number of prompts", cxxopts::value<size_t>()->default_value("1"))
-    ("dynamic_split_fuse", "Whether to use dynamic split-fuse or vLLM scheduling", cxxopts::value<bool>()->default_value("false"))
-    ("m,model", "Path to model and tokenizers base directory", cxxopts::value<std::string>()->default_value("."))
-    ("a,draft_model", "Path to assisting model base directory", cxxopts::value<std::string>()->default_value("."))
-    ("d,device", "Target device to run the model", cxxopts::value<std::string>()->default_value("CPU"))
-    ("use_prefix", "Whether to use a prefix or not", cxxopts::value<bool>()->default_value("false"))
-    ("h,help", "Print usage");
-
-    cxxopts::ParseResult result;
-    try {
-        result = options.parse(argc, argv);
-    } catch (const cxxopts::exceptions::exception& e) {
-        std::cout << e.what() << "\n\n";
-        std::cout << options.help() << std::endl;
-        return EXIT_FAILURE;
-    }
-
-    if (result.count("help")) {
-        std::cout << options.help() << std::endl;
-        return EXIT_SUCCESS;
+    if (4 != argc) {
+        throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR> <DRAFT_MODEL_DIR> '<PROMPT>'");
     }
 
-    const size_t num_prompts = result["num_prompts"].as<size_t>();
-    const bool dynamic_split_fuse = result["dynamic_split_fuse"].as<bool>();
-    const std::string model_path = result["model"].as<std::string>();
-    const std::string draft_model_path = result["draft_model"].as<std::string>();
-    const std::string device = result["device"].as<std::string>();
-    const bool use_prefix = result["use_prefix"].as<bool>();
+    ov::genai::GenerationConfig config;
+    config.max_new_tokens = 100;
+    config.num_assistant_tokens = 5;
```

Comment:
you can add a comment above that users are encouraged to use other values depending of main / draft model or even new mode, which is based on probabilities

Or add such information about 2 modes in readme.md